common/unsafe/src/main/java/org/apache/spark/unsafe/hash/Murmur3_x86_32.java

```diff
@@ -59,6 +60,18 @@ public static int hashUnsafeWords(Object base, long offset, int lengthInBytes, i
     return fmix(h1, lengthInBytes);
   }
 
+  public static int hashUnsafeBytes(MemoryBlock base, long offset, int lengthInBytes, int seed) {
```

Comment:
Same as above. I strongly feel that we should not have 2 flavours of the same method. In case of bug fixes / changes, it might happen that people forget to change in both the places. 

I know that doing it in this PR might be lot of changes and it will take time to do that in the entire codebase. Meantime, we could avoid having 2 versions of the same method.

Reply:
It makes sense. Is it better to add postfix `MB` to another version of the method?